src/metrics/gameMetrics.ts

```diff
@@ -4,4 +4,7 @@ export const gamesPlayedMetric = new PromMetricCounter({
   name: 'games_played_total',
   help: 'test',
   labelNames: ['status'],
+  labelOptions: {
+    status: ['finished', 'voided'],
```

Comment:
How about we get rid of labelNames and retrieve labelNames via Object.keys() on this guy?